app/src/helpers/test/testBuilders.ts

```diff
@@ -6,6 +6,21 @@ import {
 } from '../../types/pages/UploadDocumentsPage/types';
 import { PatientDetails } from '../../types/generic/patientDetails';
 import { SearchResult } from '../../types/generic/searchResult';
+import { UserAuth } from '../../types/blocks/userAuth';
+
+const buildUserAuth = (userAuthOverride?: Partial<UserAuth>) => {
+    const auth: UserAuth = {
+        organisations: [
+            {
+                org_name: 'PORTWAY LIFESTYLE CENTRE',
+                ods_code: 'A470',
+                role: 'DEV',
+            },
+        ],
+        authorisation_token: '111xxx222',
+    };
+    return auth;
```

Comment:
I think probably a {...userAuthOverride} is missing here?  If it is just the test is WIP then no probs.